tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/testing/index.ts

```diff
@@ -109,15 +110,33 @@ export function createTestGraphOpInfo(
 export function createTestStackFrame(
   host_name?: string,
   file_path?: string,
-  lineno?: number
+  lineno?: number,
+  function_name?: string
 ): StackFrame {
+  return {
+    host_name: host_name || 'localhost',
+    file_path: file_path || `/tmp/file_${Math.floor(Math.random() * 1e6)}.py`,
+    // `lineno` is assumed to be 1-based. So a value of 0 means use default
+    // behavior.
+    lineno: lineno || 1 + Math.floor(Math.random() * 1e3),
+    function_name:
+      function_name || `function_${Math.floor(Math.random() * 1e3)}`,
+  };
+}
+
+export function createTestStackFrameAsArray(
+  host_name?: string,
+  file_path?: string,
+  lineno?: number,
+  function_name?: string
```

Comment:
prefer an object to a builder: I don't want to refer to this code when understanding the positional argument; it is better to have the name as a key to the argument.

Reply:
Done, and changed all the call sites as well.